addons/binding/org.openhab.binding.mysensors/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MySensors Binding
+Bundle-SymbolicName: org.openhab.binding.mysensors;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ com.google.common.base,
+ com.google.common.collect,
+ com.google.gson;version="2.2.4",
```

Comment:
Do you need that specific version of gson? If not, please remove version constraint.